doc/man3/SSL_CTX_set_tlsext_servername_callback.pod

```diff
@@ -150,6 +150,27 @@ SSL_set_tlsext_host_name() returns 1 on success, 0 in case of error.
 L<ssl(7)>, L<SSL_CTX_set_alpn_select_cb(3)>,
 L<SSL_get0_alpn_selected(3)>, L<SSL_CTX_set_client_hello_cb(3)>
 
+=head1 HISTORY
+
+SSL_get_servername() historically provided some unexpected results in certain
+corner cases. This has been fixed from OpenSSL 1.1.1e. Prior to that version
+the following occurred:
+
+If the client requests a servername in an initial TLSv1.2 handshake, the
```

Comment:
I would simplify/reduce these sentences.  Perhaps just use the following construct:
```
In previous releases, when XXXXX the result would be YYYYY; it has now been changed to ZZZZZ.  {Optional second sentence with rationale.}
```

Reply:
I've reformulated those sentences along these lines.